src/curl.c

```diff
@@ -260,6 +255,18 @@ bool swupd_curl_init()
 	return true;
 }
 
+bool swupd_curl_init()
```

Comment:
Could we move this function back to `swupd_curl.h`? I could use this to check whether curl is initialized for the pack downloads without adding another function.

Reply:
Forget about the comment above, this function can stay where it is. With the changes to the asynchronous curl code, attempts to download packs over the network will fail quickly, so swupd can move on.